### ISSUE TITLE
Feature Request: Calendar view of tvshow section

### ISSUE THREAD

**Issue by inboxofjeremy** [wako-app/wako#111]
Description: As a user I want to see the last aired episodes in order of most recent too see what I missed or could be watching.

Concept: If you could use the same calendar feature that you used for the home section that shows the dates of your next episode from the list but change it to be calendar of the most recent aired of all shows. ( Kinda like a tv guide) or if there was a way to filter tvshow section by most recently aired episodes.

**Comment by JumBay** [comment]
You wanna see all tv shows even if you don't follow them?

**Comment by inboxofjeremy** [comment]
Correct, it would help see what is airing( new show just started or new
season just started as examples)

**Comment by inboxofjeremy** [comment]
If the lift is too much for a air date calendar a second thought would be to allow importing list from MDBList.
